Sources/PackageType/Module.swift

```diff
@@ -99,6 +99,28 @@ public class TestModule: SwiftModule {
     }
 }
 
+public class XcodeModule: Module {
```

Comment:
I don't think there should be a `XcodeModule` since Modules here are things which can be built.